docs/maintenance.md

```diff
@@ -31,6 +31,7 @@ Edit the following files.
 - `Makefile`
 - `e2e/Makefile`
 - `example/Makefile`
+- `example/migrate/kube-scheduler.yaml`
```

Comment:
`example/migrate/kube-scheduler.yaml` was removed, so please delete this line.

Reply:
Sorry, I deleted it.

https://github.com/topolvm/topolvm/pull/592/commits/b7a6c14de6957c9dcf8e60c56161e7e41eca6438